hack/jenkins/common.sh

```diff
@@ -301,8 +301,10 @@ if test -f "${TEST_OUT}"; then
 fi
 touch "${TEST_OUT}"
 
-if [ ! -z "${CONTAINER_RUNTIME}" ]
+if [ -z "${CONTAINER_RUNTIME}" ]
 then
+    EXTRA_START_ARGS="${EXTRA_START_ARGS} --container-runtime=docker"
```

Comment:
nit: how about not passing anything for "Docker" that way we can test that if container-runtime not passed at all it will do the right thing.

Reply:
sure